darts/models/forecasting/ensemble_model.py

```diff
@@ -26,10 +27,24 @@ class EnsembleModel(GlobalForecastingModel):
     ----------
     models
         List of forecasting models whose predictions to ensemble
+
+        .. note::
+                if all the models are probabilistic, the `EnsembleModel` will also be probabilistic.
```

Comment:
This is true for naive ensemble but not for RegressionEnsembleModel, right?

Reply:
Correct, the docstring is different in `RegressionEnsembleModel`. This note could probably be removed since `EnsembleModel` cannot be instantiated anyway.